frameworks/cassandra/src/main/dist/cassandra.yaml

```diff
@@ -63,16 +63,35 @@ unlogged_batch_across_partitions_warn_threshold: 10
 compaction_large_partition_warning_threshold_mb: 100
 request_scheduler: org.apache.cassandra.scheduler.NoScheduler
 server_encryption_options:
+{{#CASSANDRA_ENABLE_TLS}}
+    internode_encryption: all
+{{/CASSANDRA_ENABLE_TLS}}
+{{^CASSANDRA_ENABLE_TLS}}
     internode_encryption: none
-    keystore: conf/.keystore
-    keystore_password: cassandra
-    truststore: conf/.truststore
-    truststore_password: cassandra
+{{/CASSANDRA_ENABLE_TLS}}
+    keystore: node.keystore
+    keystore_password: notsecure
+    truststore: node.truststore
+    truststore_password: notsecure
+    protocol: TLSv1.2
+    cipher_suites: [TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256, TLS_RSA_WITH_AES_128_CBC_SHA256, TLS_RSA_WITH_AES_128_CBC_SHA]
 client_encryption_options:
+{{#CASSANDRA_ENABLE_TLS}}
+    enabled: true
```

Comment:
I skipped using provided values because on update, the user can provide an arbitrary value, i.e. `TRue` or `"TRUE"` via scheduler environment variables. So we probably have guarantees when user is installing service via UI, but when using the `options.json` file or updating the service there is a lots of room for error.

Reply:
Can we structure this similar to Kafka:

```
{{^CASSANDRA_ENABLE_TLS}}
enabled: false
{{/CASSANDRA_ENABLE_TLS}}
{{#CASSANDRA_ENABLE_TLS}}
enabled: true
{{#CASSANDRA_ALLOW_PLAINTEXT}}
optional: true
{{/CASSANDRA_ALLOW_PLAINTEXT}}
{{^CASSANDRA_ALLOW_PLAINTEXT}}
optional: false
{{/CASSANDRA_ALLOW_PLAINTEXT}}
... other TLS settings
{{/CASSANDRA_ENABLE_TLS}}
```